### DISCUSSION TITLE
Provide  way to export sales data to a CSV

### DISCUSSION THREAD

**Discussion by babsgosgens** [gameserverapp/Platform#962] [Suggestions]
Could we have a way to export the sales data as a CSV file?

**Comment by mvaessen** [comment]
duplicate of your own https://github.com/gameserverapp/Platform/discussions/1072

**Reply by babsgosgens** [reply]
woops. indeed it is. how about i remove this one, the other one is much more detailed.

**Reply by babsgosgens** [reply]
nm. you closed it already. <3
